source/_patterns/01-elements/input/_input.variables.scss

```diff
@@ -1,5 +1,15 @@
-$icons-path: "../../../icons/" !default;
-@use "@db-ui/base/build/scss/variables" as *;
-@use "@db-ui/base/build/scss/helpers/functions" as *;
-
 $input---color: $db-colors-secondary-enabled !default;
```

Comment:
this color doesn't seem to be in use anymore, so probably this file could get removed in total.

Reply:
updated in https://github.com/db-ui/core/pull/198/commits/9291c0e751599052a5c87dc858f92b0e719c1eaa